### PR TITLE
Changing the tooltip dot from a path to a circle helps resolve #485

### LINKED ISSUE #485: Different dots
Is it possible to draw entirely different dots, maybe even squares, or circles within circles (I work with a picky visual designer). In other words, is it possible to alter the dots beyond what is possible with CSS styling? This would also apply to the radius of the dots.

### PR DESCRIPTION
**Pull request by hambyiii**
As discussed in this thread: https://github.com/n3-charts/line-chart/pull/494 I have switched the tooltip dot from a path to a circle, so that it's radius can be altered via CSS.

### CODE REVIEW

File: src/factories/Tooltip.ts [hunk 1]
```diff
@@ -270,19 +266,17 @@ module n3Charts.Factory {
...
-            if (d.series.hasTwoKeys()) {
-              return circlePath(radius, xScale(d.row.x), yScale(d.series.axis)(d.row.y0));
-            }
-
-            return '';
-          },
-          'stroke': (d) => d.series.color
+          'r': (d) => d.series.hasTwoKeys() ? radius : null,
+          'cx': (d) => d.series.hasTwoKeys() ? xScale(d.row.x) : null,
+          'cy': (d) => d.series.hasTwoKeys() ? yScale(d.series.axis)(d.row.y1) : null,
```
Comment (lorem--ipsum) [inline src/factories/Tooltip.ts]: This should be `yScale(d.series.axis)(d.row.y0)`

**Comment by hambyiii** [comment]
Thanks for spotting that...how embarrassing. :)

**Comment by lorem--ipsum** [comment]
No worries !

**Comment by lorem--ipsum** [comment]
🍗

**Comment by lorem--ipsum** [comment]
Published 2.0.26

**Comment by hambyiii** [comment]
whoop!

**Pull request merged**